src/js/components/awardv2/shared/overview/Recipient.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Recipient.jsx
+ * Created by Jonathan Hill 11/26/19
+ **/
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+    getAwardTypeByRecordtypeCountyAndState,
+    isAwardFinancialAssistance
+} from 'helpers/awardSummaryHelper';
+import {
+    aggregateTextRecipientSection,
+    aggregateGlossaryLinks,
+    aggregateGlossaryText
+} from 'dataMapping/awardsv2/awardOverview';
+import { Glossary } from 'components/sharedComponents/icons/Icons';
+import AwardSection from '../AwardSection';
+import RecipientAddress from './RecipientAddress';
+
+
+const propTypes = {
+    recipient: PropTypes.object,
+    awardType: PropTypes.string,
+    placeOfPerformance: PropTypes.object,
+    recordType: PropTypes.number,
+    awardId: PropTypes.string
+};
+
+export default class Recipient extends Component {
```

Comment:
if we're not using state or lifecycle methods (or even if we are) lets make this a functional component. There are performance benefits to doing this and it's considered a best practice.